packages/celotool/src/lib/cloud-provider.ts

```diff
@@ -0,0 +1,44 @@
+import { createNamespaceIfNotExists } from 'src/lib/cluster'
+import { execCmd, execCmdWithExitOnFailure } from 'src/lib/cmd-utils'
+import { installAndEnableMetricsDeps, redeployTiller } from 'src/lib/helm_deploy'
+
+export interface ClusterConfig {
+  clusterName: string
+  cloudProviderName: string
+}
+
+// Configures kubectl to connect to a cluster named clusterConfig.clusterName
+export async function isContextSet(
```

Comment:
Because this function could actually change the context and only returns false if the cluster credentials aren't present, I think this name is a little misleading

Similarly-- on AKS, the name of the kubectl context is simply the name of the cluster. Is that the same on AWS?

Edit: ah I see the `--alias` flag, just wanna confirm though

Reply:
Yup, the --alias flag will set the context name to be the cluster name. Will also rename this function to be less misleading